nengo/dists.py

```diff
@@ -156,8 +156,10 @@ class UniformHypersphere(Distribution):
 
     """
 
-    def __init__(self, surface=False):
+    def __init__(self, low=0, high=1, surface=False):
```

Comment:
Docstring needs to be updated with these parameters, along with an explanation of how they effect the distribution.